noisyatom/blogs/models.py

```diff
@@ -0,0 +1,90 @@
+
+from django.conf import settings
+from django.contrib.contenttypes.models import ContentType
+from django.core.urlresolvers import reverse
+from django.db import models
+from django.db.models.signals import pre_save
+from django.utils import timezone
+from django.utils.safestring import mark_safe
+from django.utils.text import slugify
+
+from markdown_deux import markdown
+
+
+'''
+https://docs.djangoproject.com/en/1.11/ref/models/fields/
+'''
+
+class PostManager(models.Manager):
+
+	def active(self, *args, **kwargs):
+		return super(PostManager, self).filter(
+										draft=False).filter(
+													publish__lte=timezone.now())
+
+
+def upload_image_location(instcance, filename):
+	return '%s/%s'%(instcance.id, filename)
+
+
+class Post(models.Model):
+	user = models.ForeignKey(settings.AUTH_USER_MODEL, default=1)
+	title = models.CharField(max_length=200)
+	slug = models.SlugField(unique=True)
+	content = models.TextField()
+	image = models.ImageField(upload_to=upload_image_location, 
+								null=True, blank=True, 
+								height_field='height_field',
+								width_field='width_field')
+	height_field = models.IntegerField(default=0)
+	width_field = models.IntegerField(default=0)
+
+	draft = models.BooleanField(default=False)
+	publish = models.DateField(auto_now=False, auto_now_add=False)
+
+	updated = models.DateTimeField(auto_now=False, auto_now_add=True)
+	created = models.DateTimeField(auto_now=True, auto_now_add=False)
+
+	objects = PostManager()
```

Comment:
the name 'objects' means nothing ? Can you put a doc string in the PostManager class to say what it is for? I have no idea what this 'objects' is for? The name is too generic. This does not look like clean code to me?